helix-term/src/ui/editor.rs

```diff
@@ -1002,6 +1004,8 @@ impl EditorView {
         event: &MouseEvent,
         cxt: &mut commands::Context,
     ) -> EventResult {
+        cxt.editor.reset_idle_timer();
```

Comment:
```suggestion
	if event.kind != MouseEventKind::Moved{
		cxt.editor.reset_idle_timer();
	}
```

I don't think we should be triggering the idle timeout if the cursor is just moved